src/scripts/clipperUI/frontEndGlobals.ts

```diff
@@ -5,25 +5,55 @@ import {Communicator} from "../communicator/communicator";
 import {Logger} from "../logging/logger";
 import {Constants} from "../constants";
 
-export module Clipper {
-	export let injectCommunicator: Communicator;
-	export let extensionCommunicator: Communicator;
-	export let logger: Logger;
-	export let sessionId: SmartValue<string> = new SmartValue<string>();
+import {RemoteStorage} from "../storage/remoteStorage";
+import {StorageAsync} from "../storage/storageAsync";
 
-	export function getUserSessionId(): string {
-		return this.sessionId.get();
+export class Clipper {
+	private static injectCommunicator: Communicator;
+	private static extensionCommunicator: Communicator;
+
+	private static storage: StorageAsync;
+
+	// Public for convenience?
```

Comment:
is this comment intended as a exploratory TODO? because then perhaps it would be better to track as an issue